demo-shell/src/app/components/file-view/file-view.component.html

```diff
@@ -76,20 +76,28 @@ <h1>My custom toolbar</h1>
         </adf-info-drawer-tab>
 
         <adf-info-drawer-tab label="Properties">
-            <adf-content-metadata-card *ngIf="isPreset" [node]="node"
-                                       [multi]="multi"
-                                       [preset]="customPreset"
-                                       [readOnly]="isReadOnly"
-                                       [displayAspect]="showAspect"
-                                       [displayDefaultProperties]="displayDefaultProperties"
-                                       [displayEmpty]="displayEmptyMetadata"></adf-content-metadata-card>
-
-            <adf-content-metadata-card *ngIf="!isPreset" [node]="node"
-                                       [multi]="multi"
-                                       [readOnly]="isReadOnly"
-                                       [displayAspect]="showAspect"
-                                       [displayDefaultProperties]="displayDefaultProperties"
-                                       [displayEmpty]="displayEmptyMetadata"></adf-content-metadata-card>
+            <adf-content-metadata
+                *ngIf="isPreset" [node]="node"
```

Comment:
Can we move [node]="node" to next line to have all properties in column as only this one is in row.